test/fields/collections/JSON/index.tsx

```diff
@@ -10,16 +10,33 @@ type JSONField = {
 }
 
 const JSON: CollectionConfig = {
+  slug: jsonFieldsSlug,
   access: {
     read: () => true,
   },
   fields: [
     {
       name: 'json',
       type: 'json',
+      admin: {
+        jsonSchema: {
+          fileMatch: ['a://b/foo.json'],
+          schema: {
+            type: 'object',
+            properties: {
+              foo: {
+                enum: ['bar', 'foobar'],
+              },
+              number: {
+                enum: [10, 5],
+              },
+            },
+          },
+          uri: 'a://b/foo.json',
```

Comment:
If you ONLY supply a url to a valid schema, does ajv and monaco go and fetch it and run it or does somebody need to copy schemas into the config?

It would be good to document this.

Reply:
Monaco appears to at least try to fetch it I was able to get a message of an attempt:
<img width="693" alt="image" src="https://github.com/payloadcms/payload/assets/1900724/045eaaba-05cf-4920-ac13-157d10c16f68">

---

I don't think the `schema` property will get populated if a user uses just a uri.  That schema property is passed to ajv directly, so I don't expect it to validate that way.

This may be a custom fetch we have to do if only a uri is provided.